package.json

```
@@ -56,6 +56,7 @@
         "onCommand:staticWebApps.staticWebApps.action.rerun",
         "onCommand:staticWebApps.uninstallStaticWebAppsCli",
         "onCommand:staticWebApps.viewProperties",
+        "onDebug",
```

Added this to make sure our extension is activated when the user starts debugging. 

See https://code.visualstudio.com/api/references/activation-events#onDebug

---

I think you can do this instead:
```suggestion
        "onDebugResolve:pwa-chrome",
```